capabilities/loosely-coupled-teams.md

```diff
@@ -1,46 +1,46 @@
 
 ### Monoliths Can Work
 
-A common misconception is that loosely coupled teams / architecture requires the use of microservices. That's not true. Microservices can be a forcing function to creating that loose coupling, but it's not a requirement. A well-structured monolith with clearly delineated modules and strict internal contracts can simplify deployment and reduce overhead while still allowing teams to work independently on their respective components.
+A common misconception is that loosely coupled teams and architectures require the use of microservices. That's not true. While smaller, self-contained microservices can facilitate loose coupling, they are not a system requirement. A well-structured monolith with clearly delineated modules and strict internal contracts can simplify deployment and reduce overhead while still allowing teams to work independently on their respective components.
 
 ### Balancing Autonomy and Coordination
 
-Striking the right balance between team autonomy and cross-team coordination is critical. While autonomy accelerates change by minimizing dependencies, some level of coordination is necessary to ensure alignment with broader organizational objectives. Leaders must create structured communication channels and establish governance practices that allow teams to operate independently without drifting from shared goals.
+For many reasons, striking the right balance between team autonomy and cross-team coordination is critical. While autonomy accelerates change by minimizing dependencies, some level of coordination is necessary to ensure alignment with broader organizational objectives. Leaders must create structured communication channels and establish governance practices that allow teams to operate independently without drifting from shared goals.
 
 ## Supporting Practices
 
 The following is a curated list of supporting practices to consider when looking to improve your team's Loosely Coupled Teams capability. While not every practice will be beneficial in every situation, this list is meant to provide teams with fresh, pragmatic, and actionable ideas to support this capability.
 
 ### Write Contract Tests
 
-Develop and maintain contract tests that verify the interactions between services. Contract tests ensure that services adhere to agreed-upon interfaces, enabling teams to work independently without fear of breaking integrations. By writing contract tests, teams can confidently deploy changes knowing that they will not negatively impact other services, thus supporting loose coupling.
+Develop and maintain contract tests that verify the interactions between services. Contract tests ensure that all services in a system adhere to agreed-upon interfaces, which allows teams to work independently and avoid breaking integrations. By writing contract tests, teams can deploy changes with the confidence that they will not negatively impact other services, thus supporting loose coupling.
```

Comment:
I don't think we can say `all` here, because it's not always the case that an entire system is covered in contract tests. Oftentimes, only parts of the system have this type of coverage.